### PR TITLE
[CFD-104] create event page

### PR DESCRIPTION
**Pull request by helenaglow**
### Description
Created event page and also created an EventDayTab component since its not exactly the same as the week one. 🥳

Uses dummy data structures for now.

### Reason for Change
Completes CFD-104
### Type of change

- [X] New feature (non-breaking change which adds functionality)

### Testing

I tested using the general event page but you can try doing this 
```
import React from "react";
import { Text } from "react-native";
import { SafeAreaView } from "react-native-safe-area-context";
import { Stack } from "expo-router";
import Event from "./event";
import { EventListTemp } from "./event";
const event: EventListTemp = {
  name: "Event Name",
  venue: "CFD Center", 
  startTime: new Date("2023-10-09T08:00:00.000"),
  duration: 480, 
  events: [{name: "Event Name", startTime: new Date("2023-10-09T08:00:00.000"), duration: 85}, {name: "Event Name", startTime: new Date("2023-10-09T10:00:15.000"), duration: 55, venue:"Venue"}, {name: "Event Name", startTime: new Date("2023-10-09T11:00:15.000"), duration: 55, venue: "Venue"}, {name: "Event Name", startTime: new Date("2023-10-09T15:00:15.000"), duration: 90}]
  
}
const Calendar = () => {
  return (
    <SafeAreaView className="">
      <Stack.Screen options={{ title: "Calendar", headerShown: false }} />
      <Event event={event} />
    </SafeAreaView>
  );
};

export default Calendar;
```
https://github.com/uoftblueprint/centre-for-dreams/assets/113125436/cde4f3f0-8e63-4432-930b-516e9e580344

### CODE REVIEW

File: apps/expo/src/app/event.tsx
Comment (BakerWJ) [inline apps/expo/src/app/event.tsx:1]: can you create a calendar folder, then make this calendar/[id].tsx -- https://docs.expo.dev/router/create-pages/#dynamic-routes.

File: apps/expo/src/components/NavBar.tsx
Comment (BakerWJ) [inline apps/expo/src/components/NavBar.tsx:1]: this should be able to be removed once everything is moved under calendar

**Review by BakerWJ — CHANGES_REQUESTED** [review]

**Review by BakerWJ — DISMISSED** [review]

**Comment by helenaglow** [comment]
OOPs let me remove a comment rq

**Comment by helenaglow** [comment]
Sorry baker

**Review by BakerWJ — APPROVED** [review]

**Pull request merged**